internal/api/courses/assignments/submissions/proxy/submit_test.go

```diff
@@ -0,0 +1,250 @@
+package proxy
+
+import (
+	"testing"
+
+	"github.com/edulinq/autograder/internal/api/core"
+	"github.com/edulinq/autograder/internal/config"
+	"github.com/edulinq/autograder/internal/db"
+	"github.com/edulinq/autograder/internal/grader"
+	"github.com/edulinq/autograder/internal/model"
+	"github.com/edulinq/autograder/internal/timestamp"
+	"github.com/edulinq/autograder/internal/util"
+)
+
+func TestProxySubmitBase(test *testing.T) {
+	db.ResetForTesting()
+	defer db.ResetForTesting()
+
+	testSubmissions, err := grader.GetTestSubmissions(config.GetTestdataDir(), !config.DOCKER_DISABLE.Get())
+	if err != nil {
+		test.Fatalf("Failed to get test submissions in '%s': '%v'.", config.GetTestdataDir(), err)
+	}
+
+	verifySuccessfulTestSubmissions(test, testSubmissions, nil)
+}
+
+func TestProxySubmitAtDueDate(test *testing.T) {
```

Comment:
Add a simple comment (1-2 sentences) on these tests saying what you are doing and what should happen.

(Especially since the bulk of the test is handled by another function, it can be unclear the case you are testing.)